src/services/lease-service/tests/rental-rules-validator.test.ts

```diff
@@ -0,0 +1,166 @@
+import {
+  DetailedApplicantFactory,
+  LeaseFactory,
+  ListingFactory,
+} from './factory'
+import {
+  isListingInAreaWithSpecificRentalRules,
+  isHousingContractsOfApplicantInSameAreaAsListing,
+  doesApplicantHaveParkingSpaceContractsInSameAreaAsListing,
+} from '../rental-rules-validator'
+import config from '../../../common/config'
+import nock from 'nock'
+
+describe('isListingInAreaWithSpecificRentalRules', () => {
+  it('shouldReturnFalseIfListingIsNotInAreaWithSpecificRentalRules', () => {
+    const listing = ListingFactory.build({
+      districtCode: 'AREA_WHERE_RULES_DO_NOT_APPLY',
+    })
+
+    const result = isListingInAreaWithSpecificRentalRules(listing)
+
+    expect(result).toBeFalsy()
+  })
+
+  it('shouldReturnTrueIfListingIsInAreaWithSpecificRentalRules', () => {
+    const listing = ListingFactory.build({
+      districtCode: 'OXB',
+    })
+
+    const result = isListingInAreaWithSpecificRentalRules(listing)
+
+    expect(result).toBeTruthy()
+  })
+})
+
+describe('isHousingContractsOfApplicantInSameAreaAsListing', () => {
+  it('shouldReturnFalseIfNoHousingContracts', () => {
+    const detailedApplicant = DetailedApplicantFactory.build({
+      currentHousingContract: undefined,
+      upcomingHousingContract: undefined,
+    })
+
+    const result = isHousingContractsOfApplicantInSameAreaAsListing(
+      ListingFactory.build(),
+      detailedApplicant
+    )
+
+    expect(result).toBeFalsy()
+  })
+
+  it('shouldReturnFalseIfCurrentHousingContractInOtherAreaThanListing', () => {
+    const listing = ListingFactory.build({ districtCode: 'ABC' })
+    const detailedApplicant = DetailedApplicantFactory.build({
+      currentHousingContract: LeaseFactory.build({
+        residentialArea: { code: 'XYZ' },
+      }),
+    })
+
+    const result = isHousingContractsOfApplicantInSameAreaAsListing(
+      listing,
+      detailedApplicant
+    )
+
+    expect(result).toBeFalsy()
```

Comment:
Låg risk i det här fallet men med `toBeFalsy()` skulle funktionen i teorin kunna returnera `""` och passera.. :)
```suggestion
    expect(result).toBe(false)
```